privcount/traffic_model.py

```diff
@@ -276,8 +281,10 @@ def _get_inter_packet_delays(self, strm_start_ts, byte_events):
             delay = max(long(0), long(micros))
 
             # create a packet delay for each packet
-            while bw_bytes > 0:
+            split_packet_count = 0
+            while bw_bytes > 0 and split_packet_count < TrafficModel.MAX_SPLIT_PACKET_COUNT:
```

Comment:
There's still a potential issue here, and I'm not sure how to fix it: streams can handle an unlimited amount of bytes.  So if we run a traffic collection model for a long time, we will end up storing a lot of data in memory, and then slowing down when we process it.

I guess we run it on a relay and see how it goes?